epitator/date_annotator.py

```diff
@@ -78,13 +158,14 @@ def date_to_datetime_range(text, relative_base=doc.date, prefer_dates_from='past
                 date_spans.append(ne_span)
         # Regex for formatted dates
         regex = re.compile(
+            # date MonthName yyyy
```

Comment:
The SpaCy NER might be able to recognize your pattern. This regex is basically a work around for the pattern not being picked up by the current SpaCy model. I'm not concerned about picking up every pattern the NER misses since many of those errors will go away when we update the SpaCy model. This one is of particular importance because it appears frequently in ProMED articles.

I think matching dates across line breaks might be useful more often than it causes false positives, although it's a rare situation in both cases.

I'm okay with leaving out backslashes since I haven't seen dates formatted like that before.